src/clients/QBittorrent.ts

```diff
@@ -329,38 +327,51 @@ export default class QBittorrent implements TorrentClient {
 				(await this.isSubfolderContentLayout(searchee))
 					? "Subfolder"
 					: "Original";
-
-			const skipRecheckTorrent =
-				decision === Decision.MATCH_PARTIAL ? skipRecheck : true;
-
 			const formData = new FormData();
 			formData.append("torrents", buffer, filename);
-			formData.append("tags", TORRENT_TAG);
-			formData.append("category", newCategoryName);
-
-			if (autoTMM) {
-				formData.append("autoTMM", "true");
-			} else {
-				formData.append("autoTMM", "false");
-				formData.append("savepath", save_path);
-			}
+			// searchee was linked
 			if (path) {
-				formData.append("contentLayout", "Original");
-				formData.append("skip_checking", skipRecheck.toString());
-				formData.append("paused", (!skipRecheck).toString());
+				//because it was a linked file, turn off autotmm
+				//and set a save path and datacat
+				formData.append(
+					"autoTMM",
+					flatLinking && searchee.infoHash ? autoTMM : "false",
```

Comment:
why does flatLinking matter here?

Reply:
If flatLinking isn't enabled, we'd have tracker directories, which would then necessitate autotmm be off for that torrent or it'd be moved immediately. they could have it on and then we should use true or false, since they files would be flat., but if we use autotmm that would move it to the flatlining folder anyway and leave empty tracker dirs..


it's annoying.

Reply:
We warn them if they have flatLinking disabled on qbit, so I think this is proper behavior

autotmm is ignored completely and its disabled on the injected torrent if it would be in a subfolder (tracker dir) of the linkDir....because it would otherwise move if it was autotmm true

if flatlining is enabled, because we inject to one cat, it is irrelevant....let them decide based on their original torrent